### PR TITLE
runtime-rs: add comments for runtime-rs shared directory

### PR DESCRIPTION
**Pull request by Tim-0731-Hzt**
add comments for runtime-rs shared directory

Fixes:#5197
Signed-off-by: Zhongtao Hu <zhongtaohu.tim@linux.alibaba.com>

### CODE REVIEW

File: src/runtime-rs/crates/resource/src/share_fs/utils.rs [hunk 1]
```diff
@@ -41,6 +41,14 @@ pub(crate) fn share_to_guest(
...
     Ok(do_get_guest_path(target, cid, is_volume))
 }
+// Shared path handling:
+// 1. create two directories for each sandbox:
+// -. /run/kata-containers/shared/sandboxes/$sbx_id/rw/, a host/guest shared directory which is rw
+// -. /run/kata-containers/shared/sandboxes/$sbx_id/ro/, a host/guest shared directory (virtiofs source dir) which is ro
+//
+// 2. /run/kata-containers/shared/sandboxes/$sbx_id/rw/ is bind mounted readonly to /run/kata-containers/shared/sandboxes/$sbx_id/ro/, so guest cannot modify it
+//
+// 3. host-guest shared files/directories are mounted one-level under /run/kata-containers/shared/sandboxes/$sbx_id/rw/ and thus present to guest at one level under /run/kata-containers/shared/sandboxes/$sbx_id/shared/
```
Comment (liubin) [inline src/runtime-rs/crates/resource/src/share_fs/utils.rs:51]: Is `/run/kata-containers/shared/sandboxes/$sbx_id/shared/` should be `/run/kata-containers/shared/sandboxes/$sbx_id/ro/`?

Comment (liubin) [inline src/runtime-rs/crates/resource/src/share_fs/utils.rs:51]: Should `/run/kata-containers/shared/sandboxes/$sbx_id/shared/` be `/run/kata-containers/shared/containers` in the Guest?

Comment (Tim-0731-Hzt) [reply src/runtime-rs/crates/resource/src/share_fs/utils.rs:51]: I think it should be
host-guest shared files/directories are mounted one-level under `/run/kata-containers/shared/sandboxes/$sbx_id/rw/passthrough`and thus present to guest at one level under `run/kata-containers/shared/containers/passthrough`

**Review by bergwolf — APPROVED** [review]
lgtm! Thanks!

**Review by liubin — APPROVED** [review]

**Review by quanweiZhou — APPROVED** [review]

**Pull request merged**